src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -92,7 +92,7 @@ public function getItemFromIri(string $iri, array $context = [])
     public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH): string
     {
         $resourceClass = $this->getObjectClass($item);
-        $routeName = $this->routeNameResolver->getRouteName($resourceClass, false);
+        $routeName = $this->routeNameResolver->getRouteName($resourceClass, OperationType::ITEM, []);
```

Comment:
To prevent deprecation notices, you have to change these lines too:
* [IriConverter.php#L114](https://github.com/api-platform/core/blob/2139c802abd52a81af7f6f16132eea55ad081e14/src/Bridge/Symfony/Routing/IriConverter.php#L114)
* [IriConverter.php#L126](https://github.com/api-platform/core/blob/2139c802abd52a81af7f6f16132eea55ad081e14/src/Bridge/Symfony/Routing/IriConverter.php#L126)

But yes, it's a little weird to have a `trigger_error()` with an optional argument... 😄